docs/application/vstools/tools/emulator-manager.md

```diff
@@ -1,152 +1,163 @@
 # Emulator Manager
 
-You can use emulators to run your application in a virtual environment.
+After you have built your application, you need to test the application to ensure quality. To test and debug your application, you can configure a virtual environment before deploying your application. You can use emulators to run and test your application in a virtual environment. Tizen provides an emulator, Tizen Emulator Manager (Emulator Manager). It helps you to test your application in an environment similar to a real device.
 
-In order to test the application in a variety of environments, you need a variety of emulators. The Emulator Manager helps you easily create and manage emulator instances. Basically, the Emulator Manager allows you to generate emulator instances from a predefined platform and template. In addition, you can define the settings of the virtual device, such as skin, resolution, and hardware devices.
+Emulator Manager allows you to generate emulator instances from a predefined platform and template. Additionally, it helps you to define and configure settings such as skin, resolution, and hardware devices of your virtual device. 
 
-The main features of the Emulator Manager are:
+Emulator Manager helps you to:
 
-- Defining a new emulator instance or hardware profile
-- Editing an existing emulator instance or hardware profile
-- Deleting an emulator instance or hardware profile
-- Starting and stopping the emulator instance
+- Define a new emulator instance or hardware profile
+- Edit an existing emulator instance or hardware profile
+- Delete an emulator instance or hardware profile
+- Start or stop an emulator instance
 
+This section explains how to use Emulator Manager to run and test your application in a virtual environment:
```

Comment:
Section means the paragraph under the 2nd headings.

This document explains...

Reply:
Thank you for pointing this out Ms. Kang,
I agree that the word section must not have been used here.

However, I referred to the Google documentation and see that they are using “page” and “section”.
I understand that they are using “page” because for websites, we refer to pages.

However, for Tizen we have been referring to document, probably because if a reader prints, it is termed as a document.

![this_doc](https://user-images.githubusercontent.com/36875286/59914734-8268c980-940a-11e9-87dd-77d9ffb4662f.PNG)
Please guide us, shall we refer to “page” and “section” or “document” and “section”.